api/src/main/java/org/openmrs/util/TimeZoneUtil.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.util;
+
+import org.apache.commons.lang.StringUtils;
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.format.ISODateTimeFormat;
+import org.openmrs.User;
+import org.openmrs.api.context.Context;
+import org.openmrs.ui.framework.UiFrameworkConstants;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Locale;
+import java.util.TimeZone;
+
+import static org.joda.time.DateTimeZone.UTC;
+
+/**
+ * Helps provide tools to support recommended OpenMRS time zones conventions.
+ * 
+ * @see https://wiki.openmrs.org/display/docs/Time+Zones+Conventions
+ */
+public class TimeZoneUtil {
+	
+	/**
+	 * Convert a date to the client timezone, and format it, to be readable to for the user. .
```

Comment:
`readable to for`?

Reply:
Instead of sending the date to the user with the RFC3339 we send formatted with client timezone, and readable for the user, example: instead of something like 2020/01/01T16:15:10+000 we send 2020-01-01 16:15:10

Reply:
What is the meaning of `readable to for` in the above?

Reply:
Well, as a User I dont want to see dates on weird formats like RFC3339.

Reply:
Did you intend to use both `to` and `for` next to each other?

Reply:
Obviously not, sorry.